src/models/ChartData.ts

```diff
@@ -1,4 +1,5 @@
 export interface ChartDataFormat {
     date: string;
     caseCount: number;
+    caseMovingNDaysCount: number | null;
```

Comment:
`caseMovingNDaysCountCumulative` is missing here. Also I would make those two new variables optional as not every chart needs them. For example the chart that is displayed in a Popup doesn't need those variables. Without making them optional additional code needs to be added in order to populate those variables.